docs/WIP/Tutorial_Motif_only_Matrix_Profile.ipynb

```diff
@@ -0,0 +1,839 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #16.        paa = np.array([
</pre><p>I think this can be simplified to:</p>

<pre class="ql-syntax" spellcheck="false">n = len(T)
n_chunks = n // rate  # assuming we rename dsr to rate 

paa = np.mean(np.array_split(T, n_chunks), axis=1)
</pre><p> </p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/TDAmeritrade/stumpy/pull/1046/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='16'/>